Faraday_Proxy_Tools/FaradayIO/deviceconfig.py

```diff
@@ -141,9 +148,12 @@ def update_rf(self, boot_frequency_mhz, patable_byte):
 
         :return: Nothing
         """
-        freq_list = cc430radioconfig.freq0_carrier_calculation(boot_frequency_mhz)  # create_freq_list(float(boot_frequency_mhz))
+        freq_list = cc430radioconfig.freq0_carrier_calculation(float(boot_frequency_mhz))  # create_freq_list(float(boot_frequency_mhz))
```

Comment:
Added the check:

![image](https://cloud.githubusercontent.com/assets/339144/23091796/4419132a-f573-11e6-8d6a-ea736171973f.png)